meetings/2023-04-05.md

```diff
@@ -47,23 +54,99 @@
+* SC: Good question. That is what we're doing in the CG. For example WebId-Profile falls under that category. It's a client-client spec, and we need to incubate that. What we are saying in the Charter is they are not intended to be part of it or in scope. Both in terms of client-client and also being an approach towards identity. 
+* JZ: I'm thinking of a conversation I had recently with someone who has big ideas on infrastructure and the relationship between an URI and identity. Not client-client and not even server-client, but something that Solid needs to be thinking about.
+* SC: That's under the CG. New material or things that needs to be conceptually worked out, spec'ed, incubated, or needs adoption falls under CG, in general. For example, if we had a lot of implementations with a lot of user adoption out there for WebID-Profile spec, or Type Indexes, we could make an argument that we have a good sense of how it works but needs to be spec'ed, so we'd need it to be part of the Charter, but we're not there yet. The purpose of CG is to move us closer to that so that it can be picked up by WG. Hope that clarifies.
+* JZ: Yes. WG has a specific charter and CG does everything else. 
+* SC: Yes. If it's mature enough to be further worked on, it should be carried out by WG towards a recommendation. CG basically graduates specs towards that. If you look at TR homepage/index of work items, we know not all of those are as mature as they can be. There's a lot of activity, but some items are not hitting all those things sufficiently. It's not about favoritism of solutions, but makes it easier for the Charter to be accepted if there is sufficient incubation.
+* JZ: For example, WebID-Profile and Type Index, those are client-client, so if those are out of scope they will never graduate to WG.
+* SC: In this particular charter. That's one criterion for this charter (server-client) but there's other criteria for incubation.
+* JZ: That clarifies, thanks.
+* eP: I have [a good example (solid-oidc/issue#219)](https://github.com/solid/solid-oidc/issues/219). Talks about security considerations regarding Solid-OIDC issuer. This is very important because it is the foundation of security. It's an example where I think there can't be one normative statement that makes sure this is taken care of, more of a best practice informing not allowing this statement to be altered. That could be a concrete example. 
+* SC: Comment on the issue, but do not go too deep on the issue. We shouldn't get too into the considerations around that particular issue. Just have a sense of the set of issues that are intended. You can use an example in brackets, but I wouldn't highlight it as a discussion point - not the center of this issue. 
+* SC: JZ, if you could also chime in with your feedback in the issue that'd be great.
```

Comment:
```suggestion
* eP: I have [a good example (solid-oidc/issue#219)](https://github.com/solid/solid-oidc/issues/219). Talks about security considerations regarding Solid-OIDC issuer. This is very important because it is the foundation of security. It's an example where I think there can't be one normative statement that makes sure this is taken care of, more of a best practice informing not allowing this statement to be altered. That could be a concrete example. 
* SC: Comment on the issue, but do not go too deep there. We shouldn't get too into the considerations around any particular issue. Just have a sense of the set of issues that are intended. You can use an example in brackets, but I wouldn't highlight it as a discussion point — it's not the center of this issue. 
* SC: JZ, if you could also chime in with your feedback in the issue, that'd be great.
```